packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -143,6 +143,46 @@ export enum ChainId {
   StylusTestnet = 23011913
 }
 
+type ChainInfo = {
+  description: string
+  chainType: string
+  nativeCurrency: string
+}
+
+export const chainInfo: { [chainId: number]: ChainInfo } = {
```

Comment:
We definitely can make this part of `BridgeUiConfig`. We can add these for orbit chains in `orbitChainsList.ts`, and for core chains in `getBridgeUiConfigForChain`.

Then we can read everything from `getBridgeUiConfigForChain` and won't have to add config in multiple places for new Orbit chains.